composer.json

```diff
@@ -46,6 +46,7 @@
         "drupal/ckeditor_bs_grid": "2.0.7",
         "drupal/coffee": "1.2",
         "drupal/config_distro": "1.0-alpha4",
+        "drupal/config_inspector": "^2.0",
```

Comment:
Is this something you meant to add or just something to use for local development?